Docs/CommonErrors.md

```diff
@@ -0,0 +1,70 @@
+# Common Wallet Library Errors
+
+We return a Result object for every external interface that could have the potential to throw. Each public interface and what error could occur is outlined here.
+
+## Outer Errors
+### `VerifiedIdClient`
+
+`createVerifiedIdRequest`
+- `HttpError`:
+    - Code
+    - Retryable
+- `UnsupportedProtocolError`
```

Comment:
Do we need separate errors to indicate where in the flow failure happened? Supported Protocol but failure for various other reasons. For example, getting contracts/presentation request and so on.? This applies to complete/cancel calls below as well.

Reply:
Yea I think this should be where inner errors come in.